packages/celer-web-app/src/ui/frames/Home.tsx

```diff
@@ -1,40 +1,73 @@
 import { useState } from "react";
-import { LocalStorageWrapper } from "data/storage";
+import { View, Text, ShadowPropTypesIOS } from "react-native";
+import { HomePageColors, Sizes, Fonts } from "ui/styles";
+import { LocalStorageWrapper, RecentPagesWrapper } from "data/storage";
 import { EmptyObject } from "data/util";
 
+function populateRecentPages() {
+	const recentPages = RecentPagesWrapper.load();
+	return recentPages.map((url) => {
+		const link = "#/" + url;
+		return (
+			<li key={url}>
+				<a href={link}>
+					{url}
+				</a>
+			</li>
+		);
+	});
+}
+
 export const Home: React.FC<EmptyObject> = () => {
 	const [textBundle, setTextBundle] = useState(LocalStorageWrapper.load<string>("TmpBundleString", ""));
-    
+	const SITE_PADDING = "2em";
+	const MIN_WIDTH_BEFORE_COLLAPSE = 325;
+
 	return (
-		<div>
-			<h1>
-                            Home page is working in progress.
-			</h1> 
-			<p>
-                            If you are using the celer dev server, click <a href="#/dev">here</a>
-			</p>
-			<p>
-                            If you are trying to view the route, open the route URL directly. Currently only routes on github are supported. use #/gh/:user/:repo or #/gh/:user/:repo/:branch
-			</p>
-			<hr />
-			<p>
-							Paste the content of bundle.json below or 
-                            upload to view it <input type="file" onChange={(e)=>{
-					const files = e.target.files;
-					if(files?.length && files[0]){
-						const file = files[0];
-						file.text().then(text=>{
-							LocalStorageWrapper.store<string>("TmpBundleString",text);
-							setTextBundle(text);
-						});
-					}
-				}}></input>
-			</p>
-			<textarea rows={10} cols={60} value={textBundle} onChange={(e)=>{
-				LocalStorageWrapper.store<string>("TmpBundleString",e.target.value);
-				setTextBundle(e.target.value);
-			}}></textarea>
-			{textBundle && <a href="#/local">View</a>}
-		</div>
+		<View style={{ backgroundColor: HomePageColors.background }}>
+			<View nativeID="View-title" style={{ flexDirection: "row", padding: SITE_PADDING }}>
```

Comment:
What does `nativeID` do?

Reply:
nativeID is an id used by react native components. It does the same thing as id in a div.

Reply:
Is there a reason we need an id here?

Reply:
Not necessarily, there was an id there initially an I was just trying to keep it as close to the original as possible. Looking at it, nothing references that id, so I will remove it.